explorer/ast/static_scope.cpp

```diff
@@ -23,22 +24,53 @@ auto StaticScope::Add(std::string_view name, ValueNodeView entity,
     if (static_cast<int>(status) > static_cast<int>(it->second.status)) {
       it->second.status = status;
     }
+  } else {
+    if (trace_stream_->is_enabled()) {
+      *trace_stream_ << "--- declared `" << name << "` as `" << entity
+                     << "` in `" << PrintAsID(*this) << "` ("
+                     << entity.base().source_loc() << ")\n";
+    }
   }
   return Success();
 }
 
+void StaticScope::Print(llvm::raw_ostream& out) const {
+  if (ast_node_) {
+    ast_node_.value()->Print(out);
+  } else {
+    *trace_stream_ << "root";
```

Comment:
The top level is package-scope, so maybe we can say "package" here?

Reply:
Yes, that would be better.